common/lib/dependabot/metadata_finders/base/release_finder.rb

```diff
@@ -195,7 +195,7 @@ def fetch_dependency_releases
           case source.provider
           when "github" then fetch_github_releases
           # Bitbucket doesn't support releases and Azure can't list API for annotated tags
-          when "bitbucket", "azure" then []
+          when "bitbucket", "azure", "codecommit" then []
```

Comment:
Curious: does `codecommit` support releases?

A no-op is fine for now, but just wondering if this should have a TODO comment of adding future support for code commit releases, or if it's truly somethign that should be permanently a no-op... if the latter, than perhaps the comment for bitbucket/azure should also explain why codecommit was added to the no-op list.

Reply:
Codecommit does not support releases, there is no api, so the comment can be updated to reflect that as Bitbucket and Azure do.  
I am hesitant to update it though and cause the PR to need to be approved again, it took quite a while to get approved this go around and would hate for it to be moved to the back of the line again for a comment.

Reply:
I'm working on getting this merged/released today, so if you can update the comment, I'll get CI passed and get it merged in ASAP

Reply:
Done! Thank you very much!